src/components/home/Landing.tsx

```diff
@@ -1,5 +1,33 @@
+import Image from "next/image";
+import dragon from "@/public/home/dragon.webp";
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="bg-rsd-mid-blue relative w-[50%] rounded-4xl p-3 text-center">
+      <Image
+        src={dragon}
+        alt="dragon outline"
+        //add tailwind classes for the image
+        //mask clip for edges
+        className="bg-rsd-dark-blue rounded-4xl object-contain"
```

Comment:
```suggestion
        className="bg-rsd-dark-blue rounded-4xl object-cover"
```
try this and see if it fixes the corners not being covered